PlayCover/Model/PlayApp.swift

```diff
@@ -138,6 +148,28 @@ class PlayApp: BaseApp {
         Uninstaller.clearExternalCache(info.bundleIdentifier)
     }
 
+    func createAlias() {
+        do {
```

Comment:
These do/catches should be converted to one, and there is no need for the quotation mark around the error.

Reply:
Removing the quotation marks or the back slash in the error will result in a type conversion error. Log.shared.log requires type String.